spec/tree_spec.rb

```diff
@@ -1,14 +1,79 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+tree = Tree.new
+
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(Tree.is_a? Class).to eq true
+  end
+
+  context 'when grows one foot' do
+    it "equals 1" do
+      tree.age!
+      expect(tree.height).to eq(1)
+    end
+  end
+
+  context 'when ages one year' do
+    it "equals 1" do
+      expect(tree.age).to eq(1)
+    end
+  end
+
+  context 'when has no fruit' do
+    it "equals true" do
+      expect(tree.any_apples?).to eq true
+    end
+  end
+
+  context 'when has apples after 3 years' do
+    it "equals true" do
+      tree.age!
+      expect(tree.any_apples?).to be true
+    end
+  end
+
+  context 'when dies after 25 years' do
+    it "equals false" do
+      (1..13).each{ |idx| tree.age!}
+      expect(tree.dead?).to be false
+    end
   end
 end
 
-describe 'Fruit' do
+describe Fruit do
+  it 'should be a Class' do
+    expect(Fruit.is_a? Class).to eq true
+  end
 end
 
-describe 'Apple' do
+apple = Apple.new("Red", 30)
+
+describe Apple do
+  it 'should be a Class' do
+    expect(Apple.is_a? Class).to eq true
+  end
+
+  it 'is a Fruit' do
+    expect(Apple < Fruit).to be true  # expects to be inheriting from apple if it is a fruit
```

Comment:
I like this. There's a couple of other ways to do this as well.
https://stackoverflow.com/questions/23027236/how-to-test-ruby-inheritance-with-rspec

The documentation spells out your method here.
https://ruby-doc.org/core-2.3.0/Module.html#method-i-3C